Fixpostgresqlfsha3 (#13512)
hashicorp/terraform-provider-azurerm: website/docs/r/postgresql_flexible_server.html.markdown

Manages a PostgreSQL Flexible Server.

> ~> **Note** As service backend would automatically change `zone` and `standby_availability_zone` after automatic failover, so TF would cause diff. For this situation, it has to add `ignore_changes` to suppress the diff.

Reviewer: ```suggestion
~> **Note** As service backend would automatically switch `zone` and `standby_availability_zone` after automatic failover, so TF would cause diff. For this situation, users need to add `ignore_changes` to suppress the diff.
```

Author: Updated

> ~> **Note** As service backend would automatically switch `zone` and `standby_availability_zone` after automatic failover, so TF would cause diff. For this situation, users need to add `ignore_changes` to suppress the diff.

Reviewer 2: we should not be abbreviating words in the docs
```suggestion
~> **Note** As service backend would automatically switch `zone` and `standby_availability_zone` after automatic failover, so terraform would detect a config drift and attempt to apply the values in configuration. This can be prevented by using the `ignore_changes` attribute.
```

what happens after a failover if terraform is run again and tries to update the zone/standby?

Author: Once terraform is run again and tries to update the zone/standby after a automatic failover, terraform would cause two issues.

First issue:
    It would cause diff. So we have to document it here.

Second issue:
    It would try to recreate this resource. Because both zone and standby have forceNew attribute. But service team confirmed recreate operation isn't expected. So another [PR](https://github.com/hashicorp/terraform-provider-azurerm/pull/13507) I submitted to fix the issue.
